pages/home.html

```diff
@@ -21,20 +27,18 @@
     Bite-sized
   </h3>
   <p>
-    Explorations are easy to create and quick to play. They can be published in
-    the exploration gallery, shared on social media, or embedded on your
-    webpage.
+    Easy to create and quick to play, explorations can also be published in the exploration gallery, shared on social media, or embedded on your webpage.
```

Comment:
I've just collected my friends' opinions on this subject, quite interesting! Some say the original version (with 2 sentences) is clean and clear, good to go if we're writing a report. Others say the 2nd version is impressive, easy to understand, and catchy.

I collect all these opinions from my Vietnamese friends. The interesting part is that those who live in Vietnam like the second version, meanwhile those who live/study abroad prefer the original one.